src/ILLink.RoslynAnalyzer/RequiresAnalyzerBase.cs

```diff
@@ -125,6 +149,21 @@ public override void Initialize (AnalysisContext context)
 				foreach (var extraSyntaxNodeAction in ExtraSyntaxNodeActions)
 					context.RegisterSyntaxNodeAction (extraSyntaxNodeAction.Action, extraSyntaxNodeAction.SyntaxKind);
 
+				void CheckAttributeCtor (
```

Comment:
In a separate change:
This should be called `CheckAttributeInstantiation` and it should also handle the case where the instantiation sets a property on the attribute - as in that case the property setter is called and it can have RUC on it.